Source/app.py

```diff
@@ -6,19 +6,19 @@
 import time
 import shutil
 import codecs
-import tweepy
+# import tweepy
```

Comment:
I commented tweeting out, but did not remove it altogether; it is fun, but we don;t use it now. Lets resurrect it later if needed.